scripts/us_bea/states_gdp/import_industry_data_and_gen_mcf.py

```diff
@@ -148,8 +135,10 @@ def generate_mcf(self):
 
         with open('states_gdp_industry_statvars.mcf', 'w') as mcf_f:
             for naics_code in self.clean_df['NAICS'].unique():
-                code = naics_code[37:]
-                mcf_f.write(mcf_temp.format(naics=code))
+                code_title = naics_code[38:]
+                code = code_title.replace("_", "-")
```

Comment:
```suggestion
                code = code_title.replace("_", "-").replace("_", "-")
```
This is minor, but this is shorter.